apps/betterangels-backend/notes/types.py

```diff
@@ -41,7 +41,7 @@ class CreateNoteAttachmentInput:
 
 @strawberry_django.type(models.ServiceRequest, pagination=True)
 class ServiceRequestType:
-    id: auto
+    id: ID
```

Comment:
Can you explain why we're going from auto to ID?

Reply:
Is it because the schema created requires an ID vs making it optional?

Reply:
yep